src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import auxiliary.Figure;
```

Comment:
do we need this import?

Reply:
i think, yes
getArea() gets all of figures
and we need to use getColor()